From aec078e575ff994c1e7ed2dda726bf8292a4325a Mon Sep 17 00:00:00 2001
From: Jianhua Lu <lujianhua000@gmail.com>
Date: Wed, 3 Apr 2024 23:03:35 +0800
Subject: [PATCH 43/55] of: property: fix remote-endpoint parse

---
 drivers/of/property.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/of/property.c b/drivers/of/property.c
index b0633f3589de..09df55c8691e 100644
--- a/drivers/of/property.c
+++ b/drivers/of/property.c
@@ -1420,7 +1420,7 @@ static struct device_node *parse_remote_endpoint(struct device_node *np,
 	if (index > 0 || strcmp(prop_name, "remote-endpoint"))
 		return NULL;
 
-	return of_graph_get_remote_port_parent(np);
+	return of_graph_get_remote_port(np);
 }
 
 static const struct supplier_bindings of_supplier_bindings[] = {
-- 
2.48.1

